src/lib.rs

```diff
@@ -375,8 +451,73 @@ pub fn init_and_run(options: Options) -> ! {
                 );
                 ui_frame.draw_notifications_window(&notifications.borrow());
 
-                ui_frame.draw_pipeline_window(time, &mut session);
-                ui_frame.draw_operations_window(time, &mut session, DURATION_AUTORUN_DELAY);
+                if ui_frame.draw_pipeline_window(time, &mut session) {
+                    project_status.changed_since_last_save = true;
+
+                    change_window_title(&window, &project_status);
+                }
+
+                if ui_frame.draw_operations_window(time, &mut session, DURATION_AUTORUN_DELAY) {
+                    project_status.changed_since_last_save = true;
+
+                    change_window_title(&window, &project_status);
+                }
+
+                if let Some(prevent_override_status2) =
```

Comment:
Shadowing is fine. Also one is `override` and one is `overwrite`. Maybe this should be unified across the PR.